--- libopensync-plugin-evolution2-0.22/src/Makefile.am	2007-03-27 04:49:55.000000000 -0700
+++ libopensync-plugin-evolution2-0.22/src/Makefile.am.new	2008-03-13 10:54:48.000000000 -0700
@@ -4,9 +4,6 @@
 
 INCLUDES = @PACKAGE_CFLAGS@ @EPACKAGE_CFLAGS@
 
-# Please remove '-Wno-return-type' when Evo fixes the warning in their headers.
-AM_CFLAGS = -Wall -Werror -Wno-return-type
-
 config_DATA = evo2-sync
 
 opensyncheader_HEADERS = evo2_sync.h
